resume-writing/resume-writing.md

```diff
@@ -102,4 +102,8 @@ Find a partner and review each other's updated resumes. Make any necessary edits
 - [Tool: Build a Resume on Resume Genius](https://resumegenius.com/resume-formats)
 
 ### Check for Understanding
-[Submit your completed resume to be reviewed on Monster.com](https://www.monster.com/resumes/post-resume2). You get one free resume review. They claim to get back to you within 2 days with your results.
+
+- What are some points that a good resume should include. 
+- [Submit your completed resume to be reviewed on Monster.com](https://www.monster.com/resumes/post-resume2). You get one free resume review. They claim to get back to you within 2 days with your results.
```

Comment:
Since this is not really a question, could you please bring this line into ### Challenge and number the 2 challenges 1. and 2. ?

Reply:
Sorry if I was unclear... could you swap lines 100 and 108?